torch/csrc/jit/passes/dead_code_elimination.cpp

```diff
@@ -28,6 +29,12 @@ class DeadCodeEliminator {
     sweep(block, recurse);
   }
 
+  // *_once, because it should be called only once per run() call.
+  std::unordered_set<Node*> get_dead_once() {
```

Comment:
nit: can enforce this condition in code with a boolean flag for this class but reading this comment works too

Reply:
This really is not exposed anywhere other than this file, is used once, has `_once` in the name, and I don't expect it will be used for anything else, so I think we're good there 😄